tests/unit/services/gmail/test_scan.py

```
@@ -0,0 +1,356 @@
+        # Mock list to return message IDs
+        mock_list = Mock()
+        mock_list.execute.return_value = {
+            "messages": [{"id": "msg1"}, {"id": "msg2"}]
+        }
+        mock_service.users.return_value.messages.return_value.list.return_value = (
+            mock_list
+        )
+
+        # Mock batch request - callback is passed to new_batch_http_request
+        batch_callback = None
+        request_count = [0]
```

Ruff (0.14.10)

Local variable `request_count` is assigned to but never used

Remove assignment to unused variable `request_count`

(F841)